m2e-maven-runtime/org.eclipse.m2e.maven.runtime/pom.xml

```
@@ -19,7 +19,7 @@
 	</parent>
 
 	<artifactId>org.eclipse.m2e.maven.runtime</artifactId>
-	<version>3.9.100-SNAPSHOT</version>
+	<version>3.9.200-SNAPSHOT</version>
```

```suggestion
	<version>3.9.101-SNAPSHOT</version>
```
This version is meant to be aligned with the embedded Maven version, which is 3.9.1 at the moment.
Since we had a version bump in this release cycle already this is not necessary.